File: stream004/bot.js
const ws = require('ws');

const wss = new ws('ws://10.0.0.10:3434');

wss.on('message', message => {
    console.log(message)
    message = JSON.parse(message)

    if (message.message === 'hello felix') {
        wss.send(JSON.stringify({
            type: 'bot',
            name: 'felix',
            message: 'hi friend'
        }))
    }
});

File: stream004/bot.py
import asyncio
import websockets
import json

async def message():
	async with websockets.connect('ws://10.0.0.10:3434') as ws:
		while True:
			message = await ws.recv()

			message = json.loads(message)

			if message['message'] == 'hello felix':
				await ws.send(json.dumps({
					'type': 'bot',
					'name': 'felix',
					'message': 'hi friend'
				}))

asyncio.get_event_loop().run_until_complete(message())
